src/sparseml/pytorch/sparsification/quantization/helpers.py

```diff
@@ -588,44 +631,6 @@ def get_observer(
     return observer
 
 
-def fix_observer_quant_range(module: Module):
```

Comment:
we don't need this anymore because the functools.partial above handles custom qranges